### PR TITLE
blog: September newsletter

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: _posts/2017-09-26-foreman-community-newsletter---september-2017.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+### New / Updated plugins
+
+#### New plugins:
+
+The existing [foreman_userdata](https://github.com/theforeman/foreman_userdata) plugin
+has now been added to the RPM repos.
+
+#### Updated plugins:
+
+- [foreman_omaha](https://github.com/theforeman/foreman_omaha) updated to 0.0.3
```
Comment (GregSutcliffe) [inline _posts/2017-09-26-foreman-community-newsletter---september-2017.md]: minor nitpick, can you sort these? easier to scan for plugins you use that way.

**Comment by GregSutcliffe** [comment]
@orrabin weren't we going to mention the Scaleway sponsor? Also, should the survey get a paragraph of it's own, since we want to push for more replies this week?

**Comment by orrabin** [comment]
@GregSutcliffe right! does this look better?

**Comment by GregSutcliffe** [comment]
Ty @orrabin!

**Pull request merged**